cms/src/types/cms.ts

```diff
@@ -1,11 +1,12 @@
 import type { DateTime } from "luxon";
-import type { AclPermission, BaseDocumentDto, Uuid } from "./dto";
+import type { AclPermission, BaseDocumentDto, ImageFileDto, ImageUploadDto, Uuid } from "./dto";
 
 export enum DocType {
     Change = "change",
     Content = "content",
     Group = "group",
     Language = "language",
+    Image = "image",
     Media = "media",
     MediaDownload = "mediaDownload",
```

Comment:
Why is `Image` a docType in it's own right now and not a part of `Media`/`MediaType`?

Reply:
After thinking more about it I realised that image handling is quite different from other media types, so therefore I chose to keep it separate.